### PR TITLE
provide a setup document, re #221

### PR DESCRIPTION
**Pull request by lazaridiscom**
Adding basic setup instructions.

Going through `hellochain` becomes now as easy as in:

```sh
git clone https://github.com/cosmos/sdk-tutorials.git
cd sdk-tutorials/hellochain
make install
hcd
hccli
```

then continue with the "try-it-out" section.

=> zero hassle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 _[The code for these tutorials (and this site) can be found here on Github](https://www.github.com/cosmos/sdk-tutorials)_
 
+## Prerequisites
+
+After a quick [Development Environment Setup](SETUP.md), you're ready to select a tutorial from the list below.
```
Comment (marbar3778) [inline README.md]: I think it may be better to link to the official go download page instead of providing a doc that says how to install it.

Comment (lazaridiscom) [reply README.md]: No way. Setup.md contains extracted explanations (the overall task took me hours), some specific to the tutorials (e.g. windows status).

Linking simply to a download page is sloppy.

Comment (marbar3778) [reply README.md]: we have had this discussion multiple times. The decision was to link to the go install page. this is how it is done in Tendermint, Gaia and the Cosmos-sdk.

Comment (lazaridiscom) [reply README.md]: Well, if you're all proud of your result, then simply shred this PR.

Comment (marbar3778) [reply README.md]: I definitely there is a way we can work together to get this pull request merged

File: SETUP.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Development Environment Setup
+
+## Supported OS
+
+The example instructions are for Ubuntu 18.04 and for any other OS using ubuntu on a virtual machine (Windows: WSL, VirtualBox, Docker.
```
Comment (marbar3778) [inline SETUP.md]: What if we rewrite some of this to be focused toward windows but link to go install page for mac and Linux?

Comment (lazaridiscom) [reply SETUP.md]: Just merge it for now. If you (or someone else) can come up later with cli steps for mac, just add them. Similar with windows, but that seems to be high effort.

I added the link to the official download, so everyone should be happy.

(note that at this point, I'll most possibly not invest more time into this, you can take on if you want)

File: README.md (hunk 1 above)
Comment (lazaridiscom) [reply README.md]: see https://github.com/cosmos/sdk-tutorials/pull/238#discussion_r370647498

File: SETUP.md (hunk 2 above)
Comment (lazaridiscom) [reply SETUP.md]: (though maybe I give it tomorrow one more pass, I just had an idea to simplify further)

**Pull request merged**